app/styles/analysis.scss

```diff
@@ -78,5 +78,5 @@
   border-color: lighten($color-primary, 20%) !important;
 }
 .blue-checked {
-  color: rgb(0 102 255);
+  color: darken($color: $color-primary, $amount: 10);
```

Comment:
I think we don't need named arguments here.

Reply:
We should move to this syntax to be save for the future:
https://sass-lang.com/documentation/modules/color